projection/validator_delegation/delegation.go

```diff
@@ -0,0 +1,269 @@
+package validator_delegation
+
+import (
+	"fmt"
+
+	"github.com/crypto-com/chain-indexing/appinterface/rdb"
+	"github.com/crypto-com/chain-indexing/external/utctime"
+	"github.com/crypto-com/chain-indexing/projection/validator_delegation/utils"
+	"github.com/crypto-com/chain-indexing/projection/validator_delegation/view"
+	"github.com/crypto-com/chain-indexing/usecase/coin"
+)
+
+// handleDelegate - Handle create/update delegation. Return newShares and error.
+func (projection *ValidatorDelegation) handleDelegate(
+	rdbTxHandle *rdb.Handle,
+	height int64,
+	validatorAddress string,
+	delegatorAddress string,
+	amount coin.Int,
+) (coin.Dec, error) {
+
+	validatorsView := NewValidators(rdbTxHandle)
+	delegationsView := NewDelegations(rdbTxHandle)
+
+	// Get the validator
+	validator, found, err := validatorsView.FindByOperatorAddr(validatorAddress, height)
+	if err != nil {
+		return coin.ZeroDec(), fmt.Errorf("error validatorsView.FindByOperatorAddr(): %v", err)
+	}
+	if !found {
+		return coin.ZeroDec(), fmt.Errorf("error validator not found: %v, %v", validatorAddress, height)
```

Comment:
https://pkg.go.dev/fmt#hdr-Printing

Most of the time, I am just brainlessly using the `%v`, as it will print in the default format.

> %v	the value in a default format
	when printing structs, the plus flag (%+v) adds field names

Reply:
There's a subtle differences actually:
https://stackoverflow.com/questions/41159169/why-not-use-v-to-print-int-and-string

But in this case I think it should be fine as it's a string type.